.changeset/fair-beers-deny.md

```diff
@@ -0,0 +1,18 @@
+---
+"@thirdweb-dev/wallets": patch
+"@thirdweb-dev/react": patch
+---
+
+[Wallets/React] Add Blocto Wallet
+
+```javascript
+import { ThirdwebProvider, bloctoWallet } from "@thirdweb-dev/react";
+import { Polygon } from "@thirdweb-dev/chains";
+
+<ThirdwebProvider
+  activeChain={Polygon}
+  supportedWallets={[bloctoWallet({ chain: Polygon })]}
```

Comment:
Hey @q20274982, Can you update the example in changeset? `bloctoWallet()` does not take `chain` anymore

Reply:
sure, resolved in [chore: update changeset](https://github.com/thirdweb-dev/js/pull/1361/commits/ed7ad13576cd00b2933afc36937d69a35f64fae1)